src/components/comment/CommentList.jsx

```diff
@@ -8,7 +8,7 @@ const CommentList = () => {
 
   const comment = useSelector(getComment);
 
-  const commentElements = comment.map((Post, i) => (
+  const commentElements = comment.map((post, i) => (
```

Comment:
on line 7 I think you'll want the index from a post passed down so maybe ({ postIndex })and then on the line 11, it would be 
```
comments[postIndex].map((post, i) => (
do stuff
))
```